### PR TITLE
Doc: ntfs 0.5 - school_vehicle_property moved …

### PR DESCRIPTION
**Pull request by prhod**
... from trips.txt to trip_properties.txt

### CODE REVIEW

**Comment by pbougue** [comment]
Ok for me, I let @kadhikari merge this PR

And thanks for the use and feedback of git confluence doc, we corrected some points there ;)

**Comment by Hoshiyo** [comment]
Jenkins, test this please

**Comment by Hoshiyo** [comment]
Jenkins, test this please

File: documentation/ntfs/ntfs_0.5.md [hunk 1]
```diff
@@ -358,12 +358,19 @@ visual_announcement | entier (1) | Optionnel | Le véhicule dispose d'annonces v
 audible_announcement | entier (1) | Optionnel | Le véhicule dispose d'annonces sonores
 appropriate_escort | entier (1) | Optionnel | Un service d'accompagnement à bord est possible (à la montée et à la descente)
 appropriate_signage | entier (1) | Optionnel | L'affichage à bord est est claire et adapté aux personnes en déficience mentale
+school_vehicle_type | entier (2) | Optionnel | Type de transport scolaire
```
Comment (kadhikari) [inline documentation/ntfs/ntfs_0.5.md:361]: As mentioned in my last email to pascal, "school_vehicle_type" has to be replaced by "school_vehicle"

Comment (prhod) [inline documentation/ntfs/ntfs_0.5.md:361]: thanks to @kadhikari 's comment, i think the field is misnamed. "transport_type" seems like a more suitable name. Do you agree ?

Comment (kinnou02) [inline documentation/ntfs/ntfs_0.5.md:361]: :+1:

**Pull request merged**